fe/src/components/Header/HeaderLeftSide.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  HStack, Heading, Box, useColorModeValue,
+} from '@chakra-ui/react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { NAV_ITEMS } from '../../utils';
+import { Drawerbar } from '../Sidebar';
+
+export const HeaderLeftSide: React.FC = () => (
+  <HStack spacing={8} alignItems="center">
+    <Drawerbar />
+    <Heading as={Link} size="md" to="/">
+      TPT-Loane
+    </Heading>
+    <HStack as="nav" spacing={4} display={{ base: 'none', md: 'flex' }}>
+      {NAV_ITEMS.map(navItem => {
+        const { href, iconName } = navItem;
+        const newIconName = iconName;
+        return (
+          <Link key={href} to={href}>
+            <Box
+              px={2}
+              py={1}
+              rounded="md"
+              _hover={{
+                textDecoration: 'none',
+                bg: useColorModeValue('gray.200', 'gray.700'),
+              }}
+            >
+              <Box textAlign="center" minW="2em">
+                <FontAwesomeIcon icon={['fas', newIconName]} size="lg" />
```

Comment:
I dont really fancy the icons being for normal header like this:

![hmm](https://user-images.githubusercontent.com/48998862/119314560-9021cc00-bc7d-11eb-9bc5-6c9a77b29987.JPG)

Normal text links look better imho, but i'll leave that choice for you. Could be just my preference. 🙂

Reply:
agree. added labels instead of icons